src/main/java/org/kohsuke/github/GHIssue.java

```diff
@@ -180,8 +180,11 @@ private void edit(String key, Object value) throws IOException {
         new Requester(root)._with(key, value).method("PATCH").to(getApiRoute());
     }
 
-    private void editIssue(String key, Object value) throws IOException {
-        new Requester(root)._with(key, value).method("PATCH").to(getIssuesApiRoute());
+    /**
+     * @return the updated GHissue object.
+     */
+    private GHIssue editIssue(String key, Object value) throws IOException {
+        return new Requester(root)._with(key, value).method("PATCH").to(getIssuesApiRoute(), GHIssue.class);
```

Comment:
As I've learned more, I've found `Requester` supports update in place as well: 
```suggestion
new Requester(root)._with(key, value).method("PATCH").to(getIssuesApiRoute(), this);
```

I'm not sure the `GHIssue` fields that come back from patch are actually a fully populated object. 
But if they are that would save us from needing to change the method below.